app/templates/components/slide-modal.hbs

```diff
@@ -6,12 +6,16 @@
       <div class="employee-slide-btns">
         {{#each form-btns as |btn|}}
 
-        <div class="{{btn.classname}}">{{btn.title}}</div>
+        <button class="{{btn.classname}}" 
+          type ="{{btn.type}}" 
+          onclick={{if btn.isSubmit form-submit-action form-reset-action}}
```

Comment:
When we mention explicit action names, it's no longer a generic component. You can send the actionName with the button and the action as well. And you can bind the action with this button.